build.sbt

```diff
@@ -1722,6 +1722,38 @@ lazy val `distribution-manager` = project
   .dependsOn(pkg)
   .dependsOn(`logging-utils`)
 
+lazy val `bench-libs` = (project in file("tools/bench-libs"))
```

Comment:
I believe `std-bits/benchmarks` would be a better and more consistent name for a project that contains Java code and deals with Enso sources.